src/provider/zeebe/properties/MessageProps.js

```diff
@@ -37,6 +37,11 @@ export function MessageProps(props) {
   }
 
   const entries = [
+    {
```

Comment:
Since we add a specific messageName input (so to have the FEEL indicator), should we not also update https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/test/spec/provider/zeebe/MessageProps.spec.js

Reply:
Added specific tests for the message name: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/662cac38f5867c2491c7821fd2ea316908396220/test/spec/provider/zeebe/MessageProps.spec.js#L73